cirq-core/cirq/linalg/predicates.py

```diff
@@ -149,6 +149,21 @@ def is_normal(matrix: np.ndarray, *, rtol: float = 1e-5, atol: float = 1e-8) ->
     return matrix_commutes(matrix, matrix.T.conj(), rtol=rtol, atol=atol)
 
 
+def is_cptp(kraus_ops: Sequence[np.ndarray], *, rtol: float = 1e-5, atol: float = 1e-8):
```

Comment:
Do people like @viathor or @Strilanc have any strong opinions on whether we should try to support kraus ops and choi or other things one could use to specify a map ?

Reply:
If we'd like to add such support, I recommend we make a note of it in #2271 and follow up later. AFAIK there's no Choi / superoperator equivalent of #4194 in the works yet, but it would be good to have this if and when that appears.

Reply:
A reasonable way to ensure this is forward compatible with changing that choice is to force keyword args for `kraus_ops`. We can weaken it later that way if we decide to stick with it, or add other args if we want to extend.

Reply:
Good point - moved `kraus_ops` into kwargs.